### PR TITLE
Documentation emoji

### PR DESCRIPTION
**Pull request by gauravndabhade**

### CODE REVIEW

**Comment by ahnaf-zamil** [comment]
I'm leaving the review to @MaksymKarunos

File: zenora/impl/rest.py [hunk 1]
```diff
@@ -495,17 +499,17 @@ def patch_emoji(self, snowflake, emoji_id, name, roles):
             }
         return model_factory.parse_emoji(response=response, app=self)
 
-    def delete_emoji(self, snowflake, emoji_id):
+    def delete_emoji(self, snowflake: int, emoji_id: int):
         """
         Delete emoji for guild
```
Comment (MaksymKarunos) [inline zenora/impl/rest.py:504]: missing article

File: docs/build/html/_modules/index.html [hunk 2]
```diff
@@ -155,6 +155,7 @@ <h1>All modules for which code is available</h1>
 <li><a href="zenora/base/query.html">zenora.base.query</a></li>
 <li><a href="zenora/base/rest.html">zenora.base.rest</a></li>
 <li><a href="zenora/channels.html">zenora.channels</a></li>
+<li><a href="zenora/emojis.html">zenora.emojis</a></li>
```
Comment (MaksymKarunos) [inline docs/build/html/_modules/index.html:158]: The plural form of emoji is emoji, not emojiS.

File: docs/build/html/_modules/zenora/base/factory.html [hunk 3]
```diff
@@ -197,6 +197,43 @@ <h1>Source code for zenora.base.factory</h1><div class="highlight"><pre>
 <span class="sd">        zenora.users.User</span>
 <span class="sd">                Zenora user object</span>
 
+<span class="sd">        &quot;&quot;&quot;</span></div>
+
+<div class="viewcode-block" id="Factory.parse_emojis"><a class="viewcode-back" href="../../../api-reference.html#zenora.base.factory.Factory.parse_emojis">[docs]</a>    <span class="nd">@abc</span><span class="o">.</span><span class="n">abstractmethod</span>
+    <span class="k">def</span> <span class="nf">parse_emojis</span><span class="p">(</span><span class="n">response</span><span class="p">,</span> <span class="n">app</span><span class="p">):</span>
+        <span class="sd">&quot;&quot;&quot;</span>
+<span class="sd">        Parse response data from Dicord API into list of emoji objects</span>
```
Comment (MaksymKarunos) [inline docs/build/html/_modules/zenora/base/factory.html:205]: into a list of emojis.

**Review by MaksymKarunos — CHANGES_REQUESTED** [review]
First of all, congratulations on your first Pull Request. We are highly appreciative of your time and efforts you put into Zenora. We are very happy to have you.

As I mentioned in the channel, the PR is really great and it has some good quality code. The only concern is that we are missing some articles and the plural of emoji is emoji, not emojis. I tried to fix it myself, but I see that I cannot make changes to the names of the files in the PR. Please see my comments, my commit where I started fixing docstrings. 
Let me know if you have any questions!

**Comment by ahnaf-zamil** [comment]
I've checked the Sphinx docstrings and there are tons of grammatical errors. The PR is really good and the code is clean, but you need to have proper grammar in the docs so that the user wont have difficulty using the module

**Comment by gauravndabhade** [comment]
Thanks @MaksymKarunos @ahnaf-zamil for review. I will revise the documentation and update in same PR.

**Comment by ahnaf-zamil** [comment]
I'm closing this pr since there are merge conflicts

**Pull request closed without merging**